templates/genai-perf-templates/README_template

```diff
@@ -74,7 +74,7 @@ The easiest way to install GenAI-Perf is through
 Install the latest release using the following command:
 
 ```bash
-export RELEASE="{{ release }}"
+export RELEASE="yy.mm" # e.g. export RELEASE="{{ release }}"
```

Comment:
I would change this back. 
The intention here is to make the final code block on github something the user can copy and paste without modification.

Reply:
Done